cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -202,6 +202,14 @@ func getAPIServerCommand(cfg *kubeadmapi.ClusterConfiguration, localAPIEndpoint
 	if cfg.APIServer.ExtraArgs == nil {
 		cfg.APIServer.ExtraArgs = map[string]string{}
 	}
+
+	// TODO: remove in 1.21. used for smooth transition in 1.19->1.20 upgrades around the BoundServiceAccountTokenVolume feature:
```

Comment:
BoundServiceAccountTokenVolume to beta will be held for 1.21, so this change can be dropped

Reply:
done